lib/include/openamp/virtio.h

```diff
@@ -15,21 +15,38 @@ extern "C" {
 #endif
 
 /* VirtIO device IDs. */
-#define VIRTIO_ID_NETWORK    0x01UL
-#define VIRTIO_ID_BLOCK      0x02UL
-#define VIRTIO_ID_CONSOLE    0x03UL
-#define VIRTIO_ID_ENTROPY    0x04UL
-#define VIRTIO_ID_BALLOON    0x05UL
-#define VIRTIO_ID_IOMEMORY   0x06UL
-#define VIRTIO_ID_RPMSG      0x07UL /* remote processor messaging */
-#define VIRTIO_ID_SCSI       0x08UL
-#define VIRTIO_ID_9P         0x09UL
-#define VIRTIO_DEV_ANY_ID    (-1)UL
+#define VIRTIO_ID_NETWORK      1UL
+#define VIRTIO_ID_BLOCK        2UL
+#define VIRTIO_ID_CONSOLE      3UL
+#define VIRTIO_ID_ENTROPY      4UL
+#define VIRTIO_ID_BALLOON      5UL
+#define VIRTIO_ID_IOMEMORY     6UL
+#define VIRTIO_ID_RPMSG        7UL /* remote processor messaging */
+#define VIRTIO_ID_SCSI         8UL
+#define VIRTIO_ID_9P           9UL
+#define VIRTIO_ID_RPROC_SERIAL 11UL
+#define VIRTIO_ID_GPU          16UL
+#define VIRTIO_ID_INPUT        18UL
+#define VIRTIO_ID_VSOCK        19UL
+#define VIRTIO_ID_CRYPTO       20UL
+#define VIRTIO_ID_IOMMU        23UL
+#define VIRTIO_ID_MEM          24UL
+#define VIRTIO_ID_SOUND        25UL
+#define VIRTIO_ID_FS           26UL
+#define VIRTIO_ID_PMEM         27UL
+#define VIRTIO_ID_RPMB         28UL
+#define VIRTIO_ID_SCMI         32UL
+#define VIRTIO_ID_I2C_ADAPTER  34UL
+#define VIRTIO_ID_BT           40UL
+#define VIRTIO_ID_GPIO         41UL
+#define VIRTIO_DEV_ANY_ID      -1UL
```

Comment:
Could you defined all the missing devices type according to https://docs.oasis-open.org/virtio/virtio/v1.2/csd01/virtio-v1.2-csd01.html#x1-2160005

Reply:
Done.